src/Stratis.Bitcoin/Utilities/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security;
+using System.Text;
+
+namespace Stratis.Bitcoin.Utilities.Extensions
+{
+    public static class StringExtensions
+    {
+        public static SecureString ToSecureString(this string input)
```

Comment:
Should you not create a new method `FromSecureString()`?

Reply:
Good point.

Reply:
Just one more thing please add comments